### DISCUSSION TITLE
QTable with QPopupEdit

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#12493] [CLI - Electron mode] [answered]
Am i missing something here when using QPopupEdit? Is this component more of less useless? I can't seem to find a way to use it without getting a error about mutating the value from outside a mutation. Is this a way to use this with overriding the 'save' event that is fire when auto-save is enabled? That seems like it would be useful if we could redefine what is triggered when auto-save is enabled, in order to dispatch an action which then calls a mutation.

**Comment by MilosPaunovic** [comment, marked as answer]
It seems like you're having problems because of the way you've structured your code rather than the `QPopupEdit` itself. Create a computed property with getter and setter, so you won't get the error anymore.

[https://vuejs.org/guide/essentials/computed.html#writable-computed](https://vuejs.org/guide/essentials/computed.html#writable-computed)

**Reply by JokerMartini** [reply]
I'll give it a shot.
